__tests__/utils/typedData.test.ts

```diff
@@ -1,54 +1,98 @@
-import typedDataExample from '../../__mocks__/typedDataExample.json';
-import typedDataSessionExample from '../../__mocks__/typedDataSessionExample.json';
-import typedDataStructArrayExample from '../../__mocks__/typedDataStructArrayExample.json';
+import * as starkCurve from '@scure/starknet';
+
+import typedDataExample from '../../__mocks__/typedData/baseExample.json';
+import exampleBaseTypes from '../../__mocks__/typedData/example_baseTypes.json';
+import exampleEnum from '../../__mocks__/typedData/example_enum.json';
+import examplePresetTypes from '../../__mocks__/typedData/example_presetTypes.json';
+import typedDataStructArrayExample from '../../__mocks__/typedData/mail_StructArray.json';
+import typedDataSessionExample from '../../__mocks__/typedData/session_MerkleTree.json';
 import { BigNumberish, StarkNetDomain, num } from '../../src';
 import { getSelectorFromName } from '../../src/utils/hash';
 import { MerkleTree } from '../../src/utils/merkle';
 import {
+  TypedDataRevision,
   encodeType,
   encodeValue,
   getMessageHash,
   getStructHash,
   getTypeHash,
 } from '../../src/utils/typedData';
 
+const exampleAddress = '0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826';
+
 describe('typedData', () => {
   test('should get right type encoding', () => {
-    const typeEncoding = encodeType(typedDataExample.types, 'Mail');
-    expect(typeEncoding).toMatchInlineSnapshot(
+    let encoded: string;
+    encoded = encodeType(typedDataExample.types, 'Mail');
+    expect(encoded).toMatchInlineSnapshot(
       `"Mail(from:Person,to:Person,contents:felt)Person(name:felt,wallet:felt)"`
     );
-    const typeEncodingStructArr = encodeType(typedDataStructArrayExample.types, 'Mail');
-    expect(typeEncodingStructArr).toMatchInlineSnapshot(
+    encoded = encodeType(typedDataStructArrayExample.types, 'Mail');
+    expect(encoded).toMatchInlineSnapshot(
       `"Mail(from:Person,to:Person,posts_len:felt,posts:Post*)Person(name:felt,wallet:felt)Post(title:felt,content:felt)"`
     );
+    encoded = encodeType(typedDataExample.types, 'Mail', TypedDataRevision.Active);
+    expect(encoded).toMatchInlineSnapshot(
+      JSON.stringify(
+        '"Mail"("from":"Person","to":"Person","contents":"felt")"Person"("name":"felt","wallet":"felt")'
+      )
+    );
+    encoded = encodeType(typedDataStructArrayExample.types, 'Mail', TypedDataRevision.Active);
+    expect(encoded).toMatchInlineSnapshot(
+      `"\\"Mail\\"(\\"from\\":\\"Person\\",\\"to\\":\\"Person\\",\\"posts_len\\":\\"felt\\",\\"posts\\":\\"Post*\\")\\"Person\\"(\\"name\\":\\"felt\\",\\"wallet\\":\\"felt\\")\\"Post\\"(\\"title\\":\\"felt\\",\\"content\\":\\"felt\\")"`
+    );
```

Comment:
```suggestion
    expect(encoded).toMatchInlineSnapshot(
      `"Mail"("from":"Person","to":"Person","posts_len":"felt","posts":"Post*")"Person"("name":"felt","wallet":"felt")"Post"("title":"felt","content":"felt")`
    );
```
this would be easier to read

Reply:
Snapshots are autogenerated.